html/elements/a.json

```
@@ -207,7 +209,8 @@
                 "version_added": null
               },
               "webview_android": {
-                "version_added": null
+                "version_added": true,
+                "notes": "Starting in Chrome 65, cross-origin downloads are not supported on the anchor element."
```

Please use the corresponding browser name in notes

```suggestion
                "notes": "Starting in WebView 65, cross-origin downloads are not supported on the <code>&lt;a&gt;</code> element."
```